data/colors/themes/dark_high_contrast.ts

```diff
@@ -223,6 +223,12 @@ const exceptions = {
       border: get('scale.blue.5'),
     }
   },
+
+  treeViewNode: {
```

Comment:
Should we call this `treeViewItem` instead of `treeViewNode` to align with the component name (`TreeView.Item`)?
```suggestion
  treeViewItem: {
```